kcidb/monitor/spool/__init__.py

```diff
@@ -97,29 +96,48 @@ def post(self, notification, timestamp=None):
             timestamp:      An "aware" datetime.datetime object specifying the
                             notification creation time, or None to use
                             datetime.datetime.now(datetime.timezone.utc).
+            due:            An "aware" datetime.datetime object specifying the
+                            time notification should be sent out,
+                            or None to use "timestamp".
 
         Returns:
             True, if the notification was posted onto the spool,
+            or an existing notification with timestamp past the
+            picked_until time was updated,
             False, if not (it was already there).
```

Comment:
Could you change this to something higher-level and clearer, like:
```suggestion
            True, if the new notification was posted onto the spool,
            or the existing, "unpicked" notification was updated,
            False, if not (it existed and was "picked" at the moment).
```